pkg/apis/autoscaling/v2/defaults.go

```diff
@@ -115,6 +117,16 @@ func GenerateHPAScaleDownRules(scalingRules *autoscalingv2.HPAScalingRules) *aut
 	return copyHPAScalingRules(scalingRules, defaultScalingRules)
 }
 
+// BehaviorHasPolicySet checks if a given behavior contains at least one scaling
+// rule policy (for scale-up or scale-down)
+func BehaviorHasPolicySet(behavior *autoscalingv2.HorizontalPodAutoscalerBehavior) bool {
+	return behavior != nil && (ruleHasPolicySet(behavior.ScaleDown) || ruleHasPolicySet(behavior.ScaleUp))
```

Comment:
`(ruleHasPolicySet(behavior.ScaleDown) || ruleHasPolicySet(behavior.ScaleUp)`

just for my understanding, this should always be true given the validation, right?

Reply:
With my comment above this is no longer relevant. Closing this thread.